generation_tests/cargo/all_kinds/BUILD.out

```diff
@@ -5,22 +5,22 @@ load("@rules_rust//rust:defs.bzl", "rust_binary", "rust_library", "rust_test")
 rust_library(
     name = "all_kinds_lib",
     srcs = ["src/lib.rs"],
-    compile_data = [":Cargo.toml"],
+    compile_data = ["Cargo.toml"],
```

Comment:
This is based on the convention that plan source files are usually written without `:`, unlike generated targets.